README.md

```diff
@@ -36,13 +44,17 @@ Return to your original locations.  In this step all groups will act like Design
 
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
 
+*** solution: have an alarm device that wakes you with the sounds of the current weather (soothing rain, thunderstorms, wind, sunny with birds, etc.) and updates you (similar to alexa but with a soothing voice) on the days projected temperature fluctuations and meteorological daily info, hit off and an assistant will update you ***
```

Comment:
Good idea! I'm thinking a string of announcements could be released from your phone once your alarm goes off (essentially syncing the weather app to your alarm).